Source/Scene/OrthographicFrustum.js

```diff
@@ -86,15 +78,6 @@ define([
             frustum._near = frustum.near;
             frustum._far = frustum.far;
 
-            if (!frustum._useDeprecated) {
-                var ratio = 1.0 / frustum.aspectRatio;
```

Comment:
You need to keep these statements because `frustum._useDeprecated` would always be `false`.